src/Core/Framework/Migration/MigrationStep.php

```diff
@@ -34,6 +35,28 @@ public function updateDestructive(Connection $connection): void
     {
     }
 
+    public function getPlausibleCreationTimestamp(): int
+    {
+        $creationTime = $this->getCreationTimestamp();
+
+        if ($creationTime < 1 || $creationTime >= 2147483647) {
```

Comment:
can you put those magic numbers into constants? I have no clue, where the `2147483647` comes from